addons/web/static/src/legacy/js/owl_compatibility.js

```diff
@@ -245,7 +245,7 @@ odoo.define('web.OwlCompatibility', function () {
                     payload.callback(result);
                 } else {
                     throw new Error(
-                        `The service "${service}" is not present in the legacy owl environment.
+                        `The service "${payload.service}" is not present in the legacy owl environment.
```

Comment:
lol

Reply:
todo: backport in stable

Reply:
Todo extract in separate commit to !fixup on main commit (rewrite webclient in owl)